core/src/main/java/io/undertow/security/impl/InMemorySingleSignOnManager.java

```diff
@@ -28,13 +28,16 @@
 import io.undertow.server.session.Session;
 import io.undertow.server.session.SessionManager;
 import io.undertow.util.CopyOnWriteMap;
+import org.jboss.logging.Logger;
 
 /**
  * @author Stuart Douglas
  * @author Paul Ferraro
  */
 public class InMemorySingleSignOnManager implements SingleSignOnManager {
 
+    private static final Logger log = Logger.getLogger("io.undertow.client.request");
```

Comment:
This category is wrong (SSO has nothing to do with the Undertow client). In general for trace logging the name of the class should be used as the category.